tests/campaigns/test_integration.py

```diff
@@ -122,33 +123,76 @@ def test_deleted_campaign_is_returned(self):
         self.assertEqual(self.result['status'], 'Deleted')
 
 
-class TestSubscribeToSearchCampaign(BaseSearchCampaignSubscriptionTestCase):
+class TestSubscribeToSearchCampaign(BaseSubscriptionTestCase):
     def test_subscription_is_returned(self):
         self.assertTrue(self.campaign['subscription']['active'])
 
 
 class TestSubscribeToSearchCampaignNoCampaignSpecified(
-        BaseSearchCampaignSubscriptionTestCase):
+        BaseSubscriptionTestCase):
     campaign_id = -1
 
     def test_subscription_is_returned(self):
         self.assertTrue(self.campaign['subscription']['active'])
 
 
+class TestSubscribeToEverGreenSearchCampaign(BaseCampaignTestCase):
```

Comment:
I haven't seen "Green" term used elsewhere in the code. If it is not used in API, maybe we change this name to something closer to what we provide to user? "Automatic" campaign or whatever?

Reply:
Agree.  Though they used it in the discussion.